docs/usage/00-query-processing.md

```diff
@@ -0,0 +1,495 @@
+# Query Processing
+
+When a user applies filters in a search interface, the request flows through four main components:
+
+1. **UI Components** translate user interactions into SQON (Serializable Query Object Notation) filter notation
+2. **GraphQL** serves as the communication layer, combining SQON filters with field selections, pagination, and sorting parameters
+3. **Arranger Server** translates the GraphQL query into an Elasticsearch query
+4. **Elasticsearch** executes the query and returns results, which flow back through Arranger to the client
+
+This pipeline separates client-side applications from backend Elasticsearch servers.
+
+:::info **What are SQONs?**
+SQON (Serializable Query Object Notation) is Overture's filter language for communicating queries between system components. The examples below show SQONs in action, followed by detailed explanations of [what SQON is](#sqon-the-filter-language), [why it exists](#why-sqon-exists), and [how to build them](#understanding-sqon-structure).
+:::
+
+## The Processing Flow
+
+The following example demonstrates the query proecessing pipeline:
+
+![Simple Filter](../assets/query-processing.png)
+
+<details>
+<summary><b>A) View the SQON generated by the UI</b></summary>
+
+```json showLineNumbers
+{
+  "op": "and",
+  "content": [
+    {
+      "op": "in",
+      "content": {
+        "fieldName": "data.primary_site",
+        "value": ["Brain"]
```

Comment:
Could be useful to add another couple values here, to better illustrate how a wider filter would "read" (in this case `Brain OR something OR something else`)